### PR TITLE
[TS-67] Fix ability to cancel holidays by calling correct endpoint.

### PR DESCRIPTION
**Pull request by William-Unosquare**
Cancelling a holiday now calls the new `cancelHoliday` endpoint. I've added a temporary solution to confirming a holiday cancellation that should be changed later. I didn't want to modify the file too much because Paul is working in the same area of code.

### CODE REVIEW

**Review by mcromie92 — APPROVED** [review]
looks good :)

**Review by BrownMark — APPROVED** [review]

File: web-app/src/components/BookingModal/ModalStatusBanner/index.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Component } from 'react';
```
Comment (PaulTummonUnosquare) [inline web-app/src/components/BookingModal/ModalStatusBanner/index.jsx:1]: Just a small thing that if your extracting React already could you just use React.Component to make the class

**Review by PaulTummonUnosquare — COMMENTED** [review]
Yeah it all looks good!  : )

File: web-app/src/components/BookingModal/ModalStatusBanner/index.jsx (hunk 1 above)
Comment (William-Unosquare) [reply web-app/src/components/BookingModal/ModalStatusBanner/index.jsx:1]: This way of importing Component is generally the best practice.

**Pull request merged**